pool/app/system_event/entity.ml

```diff
@@ -0,0 +1,117 @@
+module Id = struct
+  include Pool_common.Id
+
+  let to_common m = m
+end
+
+module Job = struct
+  type t =
+    | GuardianCacheCleared [@name "guardiancachecleared"]
+        [@printer Utils.ppx_printer "guardiancachecleared"]
+    | TenantDatabaseAdded of Pool_database.Label.t [@name "tenantdatabaseadded"]
+        [@printer Utils.ppx_printer "tenantdatabaseadded"]
+    | TenantDatabaseUpdated of Pool_database.Label.t
+        [@name "tenantdatabaseupdated"]
+        [@printer Utils.ppx_printer "tenantdatabaseupdated"]
+    | TenantDatabaseDeleted of Pool_database.Label.t
+        [@name "tenantdatabasedeleted"]
+        [@printer Utils.ppx_printer "tenantdatabasedeleted"]
+  [@@deriving eq, show, yojson]
+
+  let read m = m |> Yojson.Safe.from_string |> t_of_yojson
+
+  let of_string str =
+    try Ok (read str) with
+    | _ -> Error Pool_common.Message.(Invalid Field.SystemEvent)
+  ;;
+
+  let to_string m = m |> yojson_of_t |> Yojson.Safe.to_string
+end
+
+type t =
+  { id : Id.t
+  ; job : Job.t
+  ; created_at : Pool_common.CreatedAt.t [@equal fun _ _ -> true]
+  ; updated_at : Pool_common.UpdatedAt.t [@equal fun _ _ -> true]
+  }
+[@@deriving eq, show]
+
+let create ?(id = Id.create ()) job =
+  { id
+  ; job
+  ; created_at = Pool_common.CreatedAt.create ()
+  ; updated_at = Pool_common.UpdatedAt.create ()
+  }
+;;
+
+module EventLog = struct
+  module ServiceIdentifier = struct
+    include Pool_common.Model.String
+
+    let field = Pool_common.Message.Field.Host
+    let schema () = schema field ()
+    let get () = () |> Unix.getpid |> CCInt.to_string
```

Comment:
This should be a combination of hostname and pid. IMO, only one of them is not enough.

Reply:
Let's try 👍